x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/WaterfallContainer/Waterfall/index.tsx

```diff
@@ -133,6 +135,10 @@ export class Waterfall extends Component<Props> {
     const itemContainerHeight = 58; // TODO: This is a nasty way to calculate the height of the svg element. A better approach should be found
     const waterfallHeight = itemContainerHeight * waterfall.orderedItems.length;
 
+    const marks = waterfall.orderedItems.filter(
+      item => item.docType === 'error' || item.docType === 'agentMark'
```

Comment:
I thought you ended up deciding not treating `agentMark` as a `docType` (because it's not) and just have it has custom payload inside transactions?